public/app/core/components/NavBar/Next/NavBarMenu.tsx

```diff
@@ -267,9 +268,21 @@ function NavItem({
         >
           <div className={styles.savedItemsMenuItemWrapper}>
             <div className={styles.iconContainer}>
-              {link.icon && <Icon name={link.icon as IconName} size="xl" />}
-              {link.img && (
-                <img src={link.img} alt={`${link.text} logo`} height="24" width="24" style={{ borderRadius: '50%' }} />
+              {link.id === 'home' ? (
+                <Branding.MenuLogo />
+              ) : (
+                <>
+                  {link.icon && <Icon name={link.icon as IconName} size="xl" />}
+                  {link.img && (
+                    <img
+                      src={link.img}
+                      alt={`${link.text} logo`}
+                      height="24"
+                      width="24"
+                      style={{ borderRadius: '50%' }}
+                    />
+                  )}
+                </>
```

Comment:
I wonder maybe if we can just scoot this out to a separate component that's just an if-statement to make this a bit easier to follow?

```tsx
const NavBarMenuLogo = ({link}) => {

if (link.id == "home") {
..
} else if (link.icon)  {
..
} etc
```

just a random thought.

Reply:
yep, have made that change. we can also extract and reuse it further in a later refactor 🤓